mpls-lab/library/conn_test_links.py

```diff
@@ -0,0 +1,54 @@
+# Krlosromero 20200321,test link connectivity
+import yaml
+import time
+from netmiko import Netmiko  
+from pprint import pprint
+from netaddr import IPNetwork
+
+# initial hosf location file --> it should be with args
+location = '../extra/hosts_file.conf'
+
+# reading yamls
```

Comment:
This comment explaining the function should be inside it as a docstring

Reply:
done